### PR TITLE
p002-p003-user-profile-and-settings-markup

### PR DESCRIPTION
**Pull request by spideyinf**
localhost 3000 p002-p003-user-profile-and-settings html 2017-10-24 17-33-52

p002 p003 user profile and settings markup

### CODE REVIEW

File: source/modules/user-profile/user-profile.pug [hunk 1]
```diff
@@ -0,0 +1,19 @@
+include ../header/header
+include ../footer/footer
+
+mixin user-profile()
+  main.user-profile
+    +header-dashboard('User Profile')
+    ul.user-profile__info-list
+      li.holder
```
Comment (emily0604) [inline source/modules/user-profile/user-profile.pug]: using loop

File: source/modules/user-profile/user-profile.pug [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      li.holder
+      li.holder
+      li.holder
+      li.holder
+      li.holder
+      li.holder
+      li.holder
+    div.user-profile__buttons
+      +button('Save', 'anchor').fr-btn-primary.mr-2
```
Comment (emily0604) [inline source/modules/user-profile/user-profile.pug]: not using mr, ml when you do task. 
These classes are only used by Backend developer

File: source/sass/abstracts/_variables.scss [hunk 3]
```diff
@@ -15,26 +15,71 @@ $font-family-primary: 'Muli', sans-serif;
...
+$white:  #fff;
+$gray-100: #f8f9fa;
+$gray-200: #e9ecef;
+$gray-300: #dee2e6;
+$gray-400: #ced4da;
+$gray-500: #adb5bd;
+$gray-600: #868e96;
+$gray-700: #495057;
+$gray-800: #343a40;
+$gray-900: #212529;
```
Comment (emily0604) [inline source/sass/abstracts/_variables.scss:43]: using Sass comman to create gray

File: source/sass/base/_typography.scss [hunk 4]
```diff
@@ -1,24 +1,42 @@
+html {
+  font-size: $font-size-base;
+
+  @include respond-to(phone-and-down) {
```
Comment (emily0604) [inline source/sass/base/_typography.scss]: try not style for html

File: source/sass/base/_typography.scss [hunk 5]
```diff
@@ -1,24 +1,42 @@
...
+    font-size: $font-size-sm;
+  }
+}
+
 body {
   font-weight: $font-weight-normal;
+  line-height: $line-height-base;
```
Comment (emily0604) [inline source/sass/base/_typography.scss]: consider customize by bootstrap

File: source/sass/layout/_header.scss [hunk 6]
```diff
@@ -0,0 +1,9 @@
+.header-dashboard {
+  display: flex;
+  justify-content: start;
+  align-items: center;
+  padding: 0 1.88rem;
```
Comment (emily0604) [inline source/sass/layout/_header.scss]: Try using $gutter of Bootstrap

**Review by emily0604 — CHANGES_REQUESTED** [review]

File: source/sass/layout/_header.scss (hunk 6 above)
Comment (spideyinf) [reply source/sass/layout/_header.scss]: Add custom $gutter: 2rem;

File: source/sass/base/_typography.scss (hunk 4 above)
Comment (spideyinf) [reply source/sass/base/_typography.scss]: Move to base/_reset.scss

File: source/sass/abstracts/_variables.scss (hunk 3 above)
Comment (spideyinf) [reply source/sass/abstracts/_variables.scss:43]: Temporary. Will research sass command for creating variables in future

File: source/modules/user-profile/user-profile.pug (hunk 1 above)
Comment (spideyinf) [reply source/modules/user-profile/user-profile.pug]: Done.

File: source/sass/base/_typography.scss (hunk 5 above)
Comment (spideyinf) [reply source/sass/base/_typography.scss]: Fixed

**Review by emily0604 — APPROVED** [review]

**Pull request merged**